docs/internals/optimizer.rst

```diff
@@ -87,7 +87,7 @@ Certain optimizer steps symbolically track the storage and memory locations. For
 information is used to compute Keccak-256 hashes that can be evaluated during compile time. Consider
 the sequence:
 
-::
+.. code-block:: assembly
 
   PUSH 32
   PUSH 0
```

Comment:
Not sure if it's a good idea. It's not x86 assembly.

Reply:
I agree with you, but the intention of using it here is to have the following differences. So I think it's more readable. 
![pic-selected-210604-0614-34](https://user-images.githubusercontent.com/36774944/120744905-52823600-c4fc-11eb-8971-6e3fdbb48c4e.png)
![pic-selected-210604-0614-12](https://user-images.githubusercontent.com/36774944/120744908-531acc80-c4fc-11eb-97e6-ca3d14a8a7c2.png)

Reply:
It's not that bad in this particular example, though still the color difference between `PUSH` and `DUP2` might make users think that they're different in some important way. In bigger samples it's even worse. Personally, I'd not highlight this one.